docs/pages/server-access/guides/auditd.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Configure SSH with the Linux Auditing System
+description: How to configure Teleport SSH with auditd (Linux Auditing System).
+h1: Linux Auditing System (auditd)
+---
+
+You can configure Teleport's SSH Service to integrate with the Linux Auditing System (auditd).
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster.
```

Comment:
```suggestion
- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster. On the Node, `teleport` must be running as a systemd service with root permissions.
```
We add this requirement later in the guide, and I think it would be good to include here so users have all the prerequisites at the top of the guide.